pkg/controller/injector/serializer_test.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package injector
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+)
+
+// Tests in this file test this package's serialize() function by "round
+// tripping." Values are serialized using the serialize() function, then are
+// deserialized using the inverse of the process used by serialize(). Comparison
+// of the original value to the output of the round trip is used to assert the
+// correcness of the serialize() function.
+
```

Comment:
I don't see this as a dead line. The comment is very general about _all_ of the tests that follow; not about any _one_ of them. I feel, therefore, that there _should_ be space separating it from what follows so it doesn't appear too specifically associated with the specific bit that follows and, by extension, less associated with the rest  of it.